cosmo_tester/resources/terraform/openstack-vm.tf.template

```diff
@@ -75,6 +75,36 @@ resource "openstack_compute_secgroup_v2" "security_group" {
     ip_protocol = "tcp"
     cidr = "${openstack_networking_subnet_v2.subnet.cidr}"
   }
```

Comment:
since the test code passes `cluster_host_ip` to `.start()`/`.join()` calls, and the managers are in the same subnet, it should work without those rules - but we need to pass `cluster_host_ip=manager1.private_ip_address`, not `.ip_address`